demos/07-sync-run-0/Kptfile

```diff
@@ -1,4 +1,4 @@
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
```

Comment:
https://github.com/GoogleContainerTools/kpt/issues/1654

Reply:
Yes, I was a bit conflicted about whether to remove this content or move it.  I am happy removing it but migrating all the content is a fairly big job so I think I'd rather prune rather than migrate and then prune.  Does that seem like an OK plan to you?

Reply:
yep, delete it please. I commented on that issue as well.